fdbserver/DataDistribution.actor.cpp

```diff
@@ -3905,7 +3905,8 @@ ACTOR Future<Void> storageServerTracker(
 		TraceEvent("StorageServerTrackerCancelled", self->distributorId)
 		    .suppressFor(1.0)
 		    .detail("Primary", self->primary)
-		    .detail("Server", server->id);
+		    .detail("Server", server->id)
+		    .detail("ErrorCode", e.code());
```

Comment:
The standard way to do this is with `.error()`:

```suggestion
		    .error(e);
```

Reply:
@xumengpanda do we want includeCancelled=true in `error`? Without it the whole TraceEvent gets suppressed if the error code is actor_cancelled.

Reply:
Yes, I think so. 
This trace event is useful to know when a storage server is no longer being tracked. That tells us when a SS left the system for what reason.

Reply:
Done